tests/test_runtime.py

```diff
@@ -131,6 +133,7 @@ async def error(*args, **kwargs):
 
         assert self.runtime.state == RuntimeStates.error, self.runtime.state
 
+    @pytest.mark.skip(reason="https://github.com/valory-xyz/open-aea/issues/408")
```

Comment:
This needs to be investigated in a separate PR asap

Reply:
that is what this PR was supposed to do address, but to no avail. At least it appears we found the causal link, but lets wait for CI to pass once more without failure on this test for additional support of that claim.